server/tracedb/tracedb.go

```diff
@@ -67,6 +67,7 @@ func (f *traceDBFactory) New(ds model.DataStore) (tdb TraceDB, err error) {
 		tdb, err = newOpenSearchDB(ds.Values.OpenSearch)
 	case openapi.SIGNAL_FX:
 		tdb, err = newSignalFXDB(ds.Values.SignalFx)
+	case openapi.AWSXRAY:
```

Comment:
Shouldn't we get the trace from X-Ray's API instead of relying on the collector here?

https://docs.aws.amazon.com/sdk-for-go/api/service/xray/#XRay.BatchGetTraces

Reply:
It's better to use the ADOT collector as it's becoming the norm for AWS and OpenTelemetry, plus, as per one of the dev teams we talked with, they will be using X-RAY (step functions) + Honeycomb. The ADOT/OTEL collectors allow us to combine multiple receivers/exporters, instead of natively supporting various data stores now.

Also, the simplest way to integrate x-ray today is through the ADOT collector, if not you'll have to set up the x-ray [daemon instance](https://docs.aws.amazon.com/xray/latest/devguide/xray-daemon.html).

I'm open to ideas tho if we want to natively use the x-ray API directly, we would need to ask users to add credentials or roles to the Tracetest instance/container so it can access it.

Reply:
Ok, so after talking to @kdhamric we are going to separate this into two things, the first one will be a recipe for Tracetest - X-Ray - ADOT collector 

And the actual settings page and BE will be the configuration to connect to the native X-Ray API

Reply:
Done 🥴